pandas/core/computation/expr.py

```diff
@@ -39,8 +36,74 @@
 
 import pandas.io.formats.printing as printing
 
+# A token value Python's tokenizer probably will never use.
+_BACKTICK_QUOTED_STRING = 100
+
+Tok = Tuple[int, str]
+
+
+def _clean_special_characters_column_name(name: str) -> str:
+    """Check if name contains any special characters, if it contains any
+    special characters the special characters will be replaced by an special
+    string and an underscore suffix is added."""
+    if not isinstance(name, str):
+        return name
+
+    if name.isidentifier() and not iskeyword(name):
+        return name
+
+    # Create a list with the special characters and their replacement.
+    # So far we only replace single character operators.
```

Comment:
Why is this restriction in place? Not trying to pester just want to be as clear as possible on where to draw the line if making changes

Reply:
That's a wrong comment indeed. Multiple character operators are composed of single operators, and therefore automatically replaced. I revisit these comments and made them clearer.